### DISCUSSION TITLE
Basic Meatloaf usage - How to emulate more than one floppy unit - Dealing with multiple floppy based games

### DISCUSSION THREAD

**Discussion by mst0mst** [idolpx/meatloaf-archived#16] [Q&A]
Greetings,

First of all, congratulations for the Meatloaf project!

Second, some basic questions about device usage:

After built an ESP32 based Meatloaf, the device works well,
its possible to connect to BBS, use the FB64 frontend, load D64 image files over the net
and over the SD card slot...

However, I still not figured out how to emulate more than one floppy unit;
such an example: I tried to play the game Turrican 2 from SD card slot.

It is a two disks based game, all goes fine loading disk 1 .d64 file;
but how to continue when the C64 prompts the message to insert the disk 2?

**Comment by ssuukk** [comment]
The functionality is not yet available, but normally Meatloaf has a web server that you will use to enable/disable drives, mount volumes and so on. Of course if anyone would rather have a physical button to swap disks (like SD2IEC) it would also be possible.
